Jamulus.pro

```diff
@@ -683,11 +683,11 @@ DISTFILES += ChangeLog \
     COPYING \
     CONTRIBUTING.md \
     README.md \
-    distributions/jamulus.desktop.in \
-    distributions/jamulus-server.desktop.in \
-    distributions/jamulus.png \
-    distributions/jamulus.svg \
-    distributions/jamulus-server.svg \
+    linux/jamulus.desktop.in \
+    linux/jamulus-server.desktop.in \
+    src/res/deb-mainicon.png \
```

Comment:
Why `deb-`?  These would be for [any freedesktop.org distribution](https://www.freedesktop.org/wiki/Specifications/desktop-entry-spec/), I think.

Reply:
Since we have mac- win- as prefix and we mainly support debian. I'm also not convinced with the renaming at all.

Reply:
I'd prefer jamulus.png jamulus.svg and jamulus-server.svg since these files are not necessarily linux only.